index.js

```diff
@@ -621,6 +621,14 @@ function hash(sess) {
  */
 
 function issecure(req, trustProxy) {
+
+  // socket is localhost
+  if (req.connection.remoteAddress === '127.0.0.1' ||
+      req.connection.remoteAddress === '::ffff:127.0.0.1'
```

Comment:
What about IPv6 localhost?

```suggestion
      req.connection.remoteAddress === '::ffff:127.0.0.1' ||
      req.connection.remoteAddress === '::1'
```

Reply:
Oh, I forgot about that you are right!